Dockerfile

```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: 2024 Stichting Health-RI
+#
+# SPDX-License-Identifier: AGPL-3.0-only
+
+FROM ckan/ckan-dev:2.10
```

Comment:
important: can you please check these changes, they may conflict with dcat-ap 3 support and ckanext-dcat 2.0.